source/funkin/play/stage/Bopper.hx

```diff
@@ -95,12 +101,7 @@ class Bopper extends StageProp implements IPlayStateScriptedClass
     if (animOffsets == null) animOffsets = [0, 0];
     if ((animOffsets[0] == value[0]) && (animOffsets[1] == value[1])) return value;
 
-    var xDiff = animOffsets[0] - value[0];
-    var yDiff = animOffsets[1] - value[1];
-
-    this.x += xDiff;
-    this.y += yDiff;
-
+    _currentAnimOffset.set(value[0], value[1]);
```

Comment:
Is there a reason you can't just use `animOffsets` in place of `_currentAnimOffset` here? The values are identical.

Reply:
Probably an oversight... will remove that.